packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -29,12 +30,15 @@ export function spawnAsync(
         return resolve();
       }
 
-      const errorLogs = stderrLogs.map(line => line.toString()).join('');
-      if (opts.stdio !== 'inherit') {
-        reject(new Error(`Exited with ${code || signal}\n${errorLogs}`));
-      } else {
-        reject(new Error(`Exited with ${code || signal}`));
-      }
+      reject(
+        new NowBuildError({
+          code: `NOW_BUILD_UTILS_SPAWN_${code}_${signal}`,
```

Comment:
The exit contains `code` _or_ `signal`, never both. So that would cause this to be for example `NOW_BUILD_UTILS_SPAWN_1_null` or `NOW_BUILD_UTILS_SPAWN_null_SIGINT` which doesn't seem great.

I suggest changing the code to a consistent value, and tacking on `exitCode` and `signal` as extra properties to the error object:

```suggestion
          code: `NOW_BUILD_UTILS_SPAWN_ERROR`,
          exitCode: code,
          signal,
```

Reply:
I decided not to add new properties at this time. Because they wouldn't go anywhere. In fact, I don't think `code` goes anywhere either. It should be visible with debug enabled when the entire error is logged instead of just `message`.